index.html

```diff
@@ -4853,6 +4853,69 @@ <h3>Issuer</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h4>Holder</h4>
+        <p>
+The value associated with the <code>holder</code> <a>property</a> is expected
+to be usable to identify the <a>holder</a> to the <a>verifier</a>.
+        </p>
+        <p>
+Often relevant metadata about the <a>holder</a>, as identified by the value of the
+<code>holder</code> <a>property</a>, is available to, or
+retrievable by, the <a>verifier</a>. For example, a <a>holder</a> can
+publish information containing the verification material used to secure
+<a>verifiable presentations</a>. This metadata is expected to be used when
+checking proofs on <a>verifiable presentations</a>. Some cryptographic
+identifiers contain all necessary metadata in the identifier itself. In those 
+cases, no additional metadata is required. Other identifiers use verifiable
+data registries where such metadata is automatically published for use
+by <a>verifiers</a>, without any additional action by the <a>holder</a>.
+        </p>
+        <p>
+See the <a data-cite="VC-IMP-GUIDE/#subject-holder-relationships"></a> and
+<a data-cite="VC-USE-CASES#user-tasks"></a> for additional examples related to <a>subject</a> and <a>holder</a>.
+        </p>
+
+        <p class="note">
+`Issuer`, `subject`, and `holder` are graph nodes which support multiple representations, 
+potentially making it complex to evaluate whether these roles are being filled
+by appropriate entities.  Validation is the process by which verifiers apply business rules to 
```

Comment:
This sentence has confusion, conflation, and / or contradiction in it. The end of the sentence says three things (`Issuer`, `subject`, and `holder`) are roles. The sentence also says that these roles are graph nodes, which conflates roles with the expression of information about the entities playing the roles. Additionally, a `subject` is neither a graph node nor a role in the VCDM, but rather it's a thing about which claims are made. An `issuer` is also not a graph node, it is a role, one an entity can perform by asserting claims about one or more subjects, etc.

I recommend we strike this sentence; I don't know what's trying to be said, it's quite mixed up, IMO, and I don't have a recommendation to fix it.

```suggestion
Validation is the process by which verifiers apply business rules to 
```

Reply:
I agree, however, we will probably need to do a second pass after this, because "validation" applies to expression of information, not to "roles".... I think its probably better to try to land this PR first, before tackling that.